libsplinter/src/protocol/authorization.rs

```diff
@@ -229,8 +229,8 @@ pub struct AuthComplete;
 /// returned.
 #[derive(Debug)]
 pub struct AuthProtocolRequest {
-    auth_protocol_min: i32,
-    auth_protocol_max: i32,
+    pub auth_protocol_min: u32,
+    pub auth_protocol_max: u32,
```

Comment:
Should these `pub` updates be in the next commit?

Reply:
updated the commits to separate adding the pub and the change to u32.